HotCommands/CommandFilter.cs

```diff
@@ -60,6 +64,8 @@ public int QueryStatus(ref Guid pguidCmdGroup, uint cCmds, OLECMD[] prgCmds, Int
                 {
                     case Constants.ToggleCommentCmdId:
                     case Constants.ExpandSelectionCmdId:
+                    case Constants.MoveMemberUpCmdId:
+                    case Constants.MoveMemberDownCmdId:
```

Comment:
What’s the best way to make this available for C# only?
Should we hook up ProviderAutoLoad on CSharpProject?
Or is there a better way?